### ISSUE TITLE
[Bug]: "Create new folder" drop-down doesn't close

### ISSUE THREAD

**Issue by schiessle** [nextcloud/server#41723]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When I try to create a new folder the drop down doesn't close as expected

Tested on c.nc.com 28RC1

### Steps to reproduce

1. Create a new folder

![image](https://github.com/nextcloud/server/assets/1589737/a946cebb-e982-4f66-8384-736521ce9487)

### Expected behavior

Drop-down closes and focus is on the input field so that I can enter the name

### Installation method

None

### Nextcloud Server version

26

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by schiessle** [comment]
Same happens for "new document", no way to give it a name. 

![image](https://github.com/nextcloud/server/assets/1589737/c49d9b11-ad82-4662-bbad-6b9efed28860)

I think in the past you could also directly chose a template and than the document opened for editing. That's probably what you want to do in 99% of all use cases when you create a new document.

**Comment by mejo-** [comment]
Probably rather 28-feedback, not 26-feedback @szaimen?

**Comment by szaimen** [comment]
> Probably rather 28-feedback, not 26-feedback @szaimen?

![image](https://github.com/nextcloud/server/assets/42591237/fdd27b35-a4a0-4396-9e20-62ffbeee6404)

**Comment by mejo-** [comment]
@schiessle did you discover this on Nextcloud 26? For me the bug you describe happens only with the new vueified Files app in Nextcloud 28.

**Comment by mejo-** [comment]
Indeed, the problem that the drop-down doesn't close seems to be solved :ok_hand: 

But wouldn't it be more intuitive if the focus would be on the new folder/document name after creating it, to be able to easily rename it? Anyway, that's another topic.
